packages/app/cypress.config.ts

```diff
@@ -21,19 +21,6 @@ export default defineConfig({
     devServer: {
       bundler: 'vite',
       framework: 'vue',
-      viteConfig: {
-        optimizeDeps: {
```

Comment:
[Vite 3.0](https://vitejs.dev/blog/announcing-vite3.html#dev-improvements) made many improvements for handling unoptimized deps. The problem this was fixing was reloads that would occur mid-test when Vite was scanning deps. From my testing, this no longer happens and we can safely remove this list.